LumpNav/LumpNav.py

```diff
@@ -228,7 +285,7 @@ def setupScene(self): #applet specific
     if not self.cauteryTipToCautery:
       self.cauteryTipToCautery=slicer.vtkMRMLLinearTransformNode()
       self.cauteryTipToCautery.SetName("CauteryTipToCautery")
-      m = self.readTransformFromSettings('CauteryTipToCautery')
+      m = self.readTransformFromSettings('Configurations/' + self.parameterNode.GetParameter('Configuration') + '/CauteryTipToCautery') 

```

Comment:
1. Yes. Application configuration is currently managed by the module widget, so you've put the code in correct location.
2. Yes
3. Yes. I think it's no problem for developers. Users should not modify these parameters.
4. Yes. To implement it, the module widget has to pass the configuration id (what is a simple machine-readable name that is used as configuration group name) to the Guidelet object. Then the Guidelet object will always try to read settings from that specific configuration group; if a value is not found then it has to be searched in the default configuration as well. Modified values should be written to the selection configuration (not the default).